nubilanding/src/components/registrarse/registrarse.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import styles from "../registrarse/styles.module.scss";
```

Comment:
Podés no es necesario ir al directorio padre para encontrar el archivo de estilos, como están en la misma carpeta podés referenciarlo así:
```suggestion
import styles from "./styles.module.scss";
```